languages/wikipron/src/codes.py

```diff
@@ -30,7 +31,13 @@
 import requests
 import requests_html
 
+# Set CUT_OFF_DATE below.
+# CUT_OFF_DATE can be no later than the date
+# at which you plan to begin scraping.
+CUT_OFF_DATE = "2019-11-01"
```

Comment:
I'm a little confused why this is given as a default here. Why not use `datetime.datetime.today().isoformat()` or something? (For our particular case where we really want to do a cut-off-date'd scrape we can just search-and-replace on` languages.json`, right?)

Reply:
For some reason I interpreted #68 as “Language entries in languages.json should contains all config options that do not use default values set in config.py.” 

So I guess I wasted some time doing that. It would be simpler if the language entries looked like they were before: 
```
    "est": {
        "iso639_name": "Estonian",
        "wiktionary_name": "Estonian",
        "wiktionary_code": "et",
        "casefold": true
    },
```
Or this if a dialect is used:
```
    "eng": {
        "iso639_name": "English",
        "wiktionary_name": "English",
        "wiktionary_code": "en",
        "casefold": true,
        "dialect": {
            "us": "US | General American",
            "uk": "UK | Received Pronunciation"
        }
    },
```

And then `config_settings` in `scrape.py` would handle setting `no_stress`, `no_syllable_boundaries`, and `cut_off_date`, since they are always the same. Then we can just set `cut_off_date` with `datetime` in `scrape.py`.